sw/device/silicon_creator/lib/boot_svc/boot_svc_next_boot_bl0_slot.h

```diff
@@ -0,0 +1,105 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_BOOT_SVC_NEXT_BOOT_BL0_SLOT_H_
+#define OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_BOOT_SVC_NEXT_BOOT_BL0_SLOT_H_
+
+#include <stdint.h>
+
+#include "sw/device/lib/base/macros.h"
+#include "sw/device/silicon_creator/lib/base/chip.h"
+#include "sw/device/silicon_creator/lib/boot_svc/boot_svc_header.h"
+#include "sw/device/silicon_creator/lib/error.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
+enum {
+  /*
+   * Encoding generated with
+   * $ ./util/design/sparse-fsm-encode.py -d 6 -m 2 -n 32 \
+   *     -s 2121036560 --language=c
+   *
+   * Minimum Hamming distance: 14
+   * Maximum Hamming distance: 14
+   * Minimum Hamming weight: 11
+   * Maximum Hamming weight: 13
+   */
+  kBootSvcNextBootBl0SlotA = 0x08c0d499,
+  kBootSvcNextBootBl0SlotB = 0x7821e03a,
+
+  /*
+   *  Encoding generated with
+   *  $ ./util/design/sparse-fsm-encode.py -d 6 -m 2 -n 32 \
+   *      -s 3290097361 --language=c
+   *
+   * Minimum Hamming distance: 15
+   * Maximum Hamming distance: 15
+   * Minimum Hamming weight: 16
+   * Maximum Hamming weight: 19
+   */
+  kBootSvcNextBl0SlotReqType = 0xe1edf546,
+  kBootSvcNextBl0SlotResType = 0x657051be,
```

Comment:
`boot_svc_empty.h` currently defines `kBootSvcEmptyType = 0xb4594546`. WDYT about consolidating all these definitions into `boot_svc_header.h` and regenerating all the constants to ensure they are have a sufficiently high Hamming distance?

Reply:
Yeah, @dmcardle raised the same point earlier. Let's keep them separate for now. I don't think there is a reason to regenerate them but we should add a unit test that enforces a minimum hamming distance of 6 to each other similar to the test in `error_unittest.cc`.